tests/e2e/e2e_lsm_test.go

```diff
@@ -0,0 +1,204 @@
+package e2e
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	gov "github.com/cosmos/cosmos-sdk/x/gov/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+)
+
+func (s *IntegrationTestSuite) testLSM() {
+	chainEndpoint := fmt.Sprintf("http://%s", s.valResources[s.chainA.id][0].GetHostPort("1317/tcp"))
+
+	validatorA := s.chainA.validators[0]
+	validatorB := s.chainB.validators[0]
+	validatorAAddr := validatorA.keyInfo.GetAddress()
+	validatorBAddr := validatorB.keyInfo.GetAddress()
```

Comment:
looks like validatorB is only used as the recipient of the ibc transfer. If it's easy enough to create an arbitrary account, it might be a little clear to instead call this like `transferRecipientAddr`

Similarly for the bank send (`sendRecipientAddr`)

Reply:
Okay

Reply:
This is sent to validatorBAddr in case it requires to return back to original chain.

Reply:
For now, updated to like this and remove this line
```
	sendRecipientAddr := s.chainB.validators[0].keyInfo.GetAddress()
	s.sendIBC(s.chainA, 0, validatorAAddr.String(), sendRecipientAddr.String(), ibcTransferAmount.String(), standardFees.String(), "memo")
```